### DISCUSSION TITLE
How to delete a friendly name?

### DISCUSSION THREAD

**Discussion by ramanraja** [arendst/Tasmota#20729] [Q&A] [answered]
I configure my Tasmota devices using their web URL with a RESTful API tool as a mobile app. 
To set up an Alexa name for a relay, I use the FriendlyName command

http://192.168.0.131/cm?cmnd=FriendlyName1%20motor

But if I want to delete the friendly name, I cannot just supply a blank name:

http://192.168.0.131/cm?cmnd=FriendlyName1

Tasmota interprets this as a request for _information_, and just returns the previously stored name.

How to delete the Alexa name and make it blank? I can easily do this using Tasmota's web UI, but the UI is locked for general users. I only want to access it through an HTTP GET call.

**Comment by sfromis** [comment, marked as answer]
The usual syntax for removing string settings is to supply an empty string, like:
`FriendlyName2 "`
Or you can reset it to firmware default:
`FrienclyName2 1`

**Reply by ramanraja** [reply]
Thanks. Using double quotes worked for me:
FriendlyName2 ""

**Reply by sfromis** [reply]
Yeah, that also works, in addition to the shorter form I mentioned. A single double quote instead of two.
